Presenter.ControlPanel/App.xaml.cs

```diff
@@ -24,9 +24,17 @@ protected override void OnStartup(StartupEventArgs e)
 
         private void Application_Exit(object sender, ExitEventArgs e)
         {
-            if (ScreenManager.Instance.Presentations == null) return;
+             if (ScreenManager.Instance.Presentations.First().Value == null)
+            {
+                return;
+            }
             foreach (var process in ScreenManager.Instance.Presentations.Values)
```

Comment:
foreach (var process in ScreenManager.Instance.Presentations.Values.Where(p => !p.HasExited))
{
   process.Kill();
}